locale/defaultMessages.json

```diff
@@ -6378,6 +6381,9 @@
     "context": "submit button",
     "string": "Add manual capture"
   },
+  "aGx3Z5": {
+    "string": "Created {formattedDate}"
+  },
```

Comment:
suggestion: I think we already have similar strings in our translations, we could reuse them instead of creating new ;)

Reply:
I would be careful with that, because with reusing, you lose context.